nncf/tensorflow/layers/wrapper.py

```diff
@@ -202,6 +203,16 @@ def _apply_ops(self, training):
             self.set_layer_weight(weight_attr, layer_weight)
 
     def registry_weight_operation(self, weights_attr: str, op: NNCFOperation):
+        # For BatchNormalization layers:
+        # If layer.scale=False then gamma parameter is fused - no need to apply the mask to it
+        if isinstance(self.layer,
+                      (tf.keras.layers.BatchNormalization, tf.keras.layers.experimental.SyncBatchNormalization)) \
+                and not self.layer.scale \
+                and weights_attr == 'gamma':
+            nncf_logger.debug('Fused gamma parameter encountered in BatchNormalization layer. '
```

Comment:
It looks like `NNCFWrapper.registry_weight_operation` is the wrong place to skip registration operation. Please handle it in the algorithm.

Reply:
Check it now please